services/auth-service/pkg/config/defaults/defaultconfig.go

```diff
@@ -0,0 +1,87 @@
+package defaults
+
+import (
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+	"github.com/owncloud/ocis/v2/ocis-pkg/structs"
+	"github.com/owncloud/ocis/v2/services/auth-service/pkg/config"
+)
+
+// FullDefaultConfig returns a fully initialized default configuration
+func FullDefaultConfig() *config.Config {
+	cfg := DefaultConfig()
+	EnsureDefaults(cfg)
+	Sanitize(cfg)
+	return cfg
+}
+
+// DefaultConfig returns a basic default configuration
+func DefaultConfig() *config.Config {
+	return &config.Config{
+		Debug: config.Debug{
+			Addr:   "127.0.0.1:9169",
+			Token:  "",
+			Pprof:  false,
+			Zpages: false,
+		},
+		GRPC: config.GRPCConfig{
+			Addr:      "127.0.0.1:9199",
+			Namespace: "com.owncloud.api",
+			Protocol:  "tcp",
+		},
+		Service: config.Service{
+			Name: "auth-service",
+		},
+		Reva: shared.DefaultRevaConfig(),
+		ServiceAccount: config.ServiceAccount{
+			ServiceAccountID:     "service-user-id",
+			ServiceAccountSecret: "secret-string",
```

Comment:
We shouldn't be setting a default secret here. That should be handled by `ocis init`

Reply:
Wouldn't that bring problems with existing installations? The `ocis.yaml` file is not rewritten when running ocis init again...

Reply:
I added it in ocis init and removed the hardcoded values. I like not having hardcoded secrets :+1: But we will have breaking changes for existing installations.

Should we fail hard if the `OCIS_SERVICE_ACCOUNT_SECRET` is not set?